### ISSUE TITLE
uWebSockets library version needs to be updated to v0.13.0

### ISSUE THREAD

**Issue by WesThorburn** [Microsoft/vcpkg#738]
Currently, vcpkg uses v0.12.0 of the uWebSockets library, unfortunately, that version has a Windows bug outlined here in [Issue #351](https://github.com/uWebSockets/uWebSockets/issues/351). This bug was fixed in v0.13.0, however Windows users installing the uWebSockets library using Vcpkg will still encounter the bug as seen in [Issue #514](https://github.com/uWebSockets/uWebSockets/issues/514)

**Comment by WesThorburn** [comment]
I've had a crack at updating it to v0.13.0 myself. I updated the uWebSockets library version number to 0.13.0 in ports/uwebsockets/portfile.cmake as well as ports/uwebsockets/CONTROL, then after removing uWebSockets and re-installing it, I'm getting the following error:
```
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:14 (message):
Command failed: C:/src/vcpkg/downloads/cmake-3.8.0-rc1-win32-x86/bin/cmake.e
xe;C:/src/vcpkg/buildtrees/uwebsockets/src/uwebsockets-0.13.0;-DBUILD_SHARED_LIB
S=ON;-DCMAKE_CXX_FLAGS= /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP;-DCMAKE_C_FL
AGS= /DWIN32 /D_WINDOWS /W3 /utf-8 /MP;-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON;-DC
MAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON;-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_
REGISTRY=ON;-DBoost_COMPILER=-vc140;-DCMAKE_CXX_FLAGS_RELEASE=/MD /O2 /Oi /Gy /D
NDEBUG /Zi;-DCMAKE_C_FLAGS_RELEASE=/MD /O2 /Oi /Gy /DNDEBUG /Zi;-DCMAKE_SHARED_L
INKER_FLAGS_RELEASE=/DEBUG /INCREMENTAL:NO /OPT:REF /OPT:ICF;-DCMAKE_EXE_LINKER_
FLAGS_RELEASE=/DEBUG /INCREMENTAL:NO /OPT:REF /OPT:ICF;-G;Visual Studio 14 2015
Win64;-DCMAKE_VERBOSE_MAKEFILE=ON;-DCMAKE_BUILD_TYPE=Release;-DCMAKE_TOOLCHAIN_F
ILE=C:/src/vcpkg/triplets/x64-windows.cmake;-DCMAKE_PREFIX_PATH=C:/src/vcpkg/ins
talled/x64-windows;-DCMAKE_INSTALL_PREFIX=C:/src/vcpkg/packages/uwebsockets_x64-
windows
    Working Directory: C:/src/vcpkg/buildtrees/uwebsockets/x64-windows-rel
    See logs for more information:
      C:\src\vcpkg\buildtrees\uwebsockets\config-x64-windows-rel-out.log
      C:\src\vcpkg\buildtrees\uwebsockets\config-x64-windows-rel-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_cmake.cmake:95 (vcpkg_execute_required_process)
  ports/uwebsockets/portfile.cmake:16 (vcpkg_configure_cmake)
  scripts/ports.cmake:73 (include)


Error: Building package uwebsockets:x64-windows failed with: BUILD_FAILED
```
What am I doing wrong here?

**Comment by alexkaratarakis** [comment]
It appears that the CMakeLists.txt was changed a bit in the new version, so the patch could not be applied.
I fixed it and updated to v0.13.0
